String founder 
Int32 age_of_company
Int32 number_of_employee
